components/desktop/gnumeric/Makefile

```diff
@@ -36,12 +36,17 @@ PATH=$(PATH.gnu)
 
 CONFIGURE_BINDIR.64 = $(CONFIGURE_BINDIR.32)
 
+COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D))
+
+PKG_CONFIG_BIN_64 = /usr/bin/$(MACH64)/pkg-config
```

Comment:
Do we really need it? 
When building 64-bit, PKG_CONFIG_PATH is set to /usr/lib/amd64/pkgconfig.